README.md

```diff
@@ -33,3 +35,14 @@ To know about the available options run the command:
 To test changes, enter the `tests` directory and run the following command:
 
     pytest tests.py
+    
+    
+## Output
+
+The program can provide various graphs upon request (wherever possible)
+For example:
+![img2](https://user-images.githubusercontent.com/84586244/145166287-5643a572-ac17-4795-bdc6-43ab82a6274c.jpeg)
+![img3](https://user-images.githubusercontent.com/84586244/145166324-194aad63-d901-41fe-bb28-9f00f465ce71.jpeg)
+![img4](https://user-images.githubusercontent.com/84586244/145166146-c8186b67-8f9b-4572-8f20-329fdb3bcbbc.jpeg)
```

Comment:
I think we could make this look a bit better by centering the images a bit.
A suggestion: 
```md
<p align="center"> 
<img src="https://user-images.githubusercontent.com/84586244/145166287-5643a572-ac17-4795-bdc6-43ab82a6274c.jpeg" />
<img src="https://user-images.githubusercontent.com/84586244/145166324-194aad63-d901-41fe-bb28-9f00f465ce71.jpeg" />
<img src="https://user-images.githubusercontent.com/84586244/145166146-c8186b67-8f9b-4572-8f20-329fdb3bcbbc.jpeg" />
</p>
```